h/schemas/admin_group.py

```diff
@@ -22,17 +22,23 @@
 )
 
 
-def user_exists_validator_factory(user_svc):
```

Comment:
Rather than passing the validator as an argument to the schema constructor, I'm using colander's "[deferred values](https://docs.pylonsproject.org/projects/colander/en/latest/binding.html)" facility to pass values only known at runtime to the validation and widget preparation logic via `schema.bind`.

I'm doing this because AFAICS it is the "standard" way to handle this need in colander.